src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs

```diff
@@ -79,6 +84,29 @@ public void StartMigrationWizard(BoundSonarQubeProject oldBinding)
             }
         }
 
+        private Action GetTfVcHelpAction()
+        {
+            Action helpOp;
+
+            // We don't have an existing component that detects whether user is using Tfvc.
+            // However, we can tell if they are using git, which we expect to be the majority
+            // of users. So to reduce the noise, we won't show the Tfvc help for git users.
```

Comment:
This comment says it all: quick and dirty hack to reduce the noise for the majority of users.